nagios/tests/test_e2e.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+
+@pytest.mark.e2e
+def test_e2e(sts_agent_check, instance):
+    # This integration is based on tailing logs but because there is an agent running
```

Comment:
I dont' understand this comment, can you rephrase it ?

Reply:
This is file copied from DataDog repo.
https://github.com/DataDog/integrations-core/blob/master/nagios/tests/test_e2e.py

Reply:
Datadog had problem with running tests because of Tailer missing its offset when reading log files. 
We encountered the same problem. 

Tailing logs works only when agent is running and starting check. If you invoke check manually Tailer will miss the offset and won't be able to parse logs.

Would this remark clarify it?

Reply:
The new message is indeed clearer

Reply:
As agreed yesterday `test_e2e.py` removed.